ts/features/wallet/paypal/PaypalCard.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from "react";
+import * as pot from "italia-ts-commons/lib/pot";
+import { Image, StyleSheet } from "react-native";
+import { View } from "native-base";
+import { connect } from "react-redux";
+import BaseCardComponent from "../component/card/BaseCardComponent";
+import paypalLogoExt from "../../../../img/wallet/payment-methods/paypal-logo.png";
+import paypalLogoMin from "../../../../img/wallet/cards-icons/paypal.png";
```

Comment:
```suggestion
import paypalLogoMin from "../../../../img/wallet/cards-icons/paypal_card.png";
```

Reply:
this is the right asset (same used in the preview card)

Reply:
![Simulator Screen Shot - iPhone 12 - 2021-11-16 at 15 14 11](https://user-images.githubusercontent.com/822471/142001292-44aafb38-82df-4f8d-bc89-756c947d05bd.png)
with this 4a63a89bf6006729224551b199993d83de04adb2